devito/operator/profiling.py

```diff
@@ -453,7 +453,7 @@ def add_glb_vanilla(self, key, time):
         if not self.input:
             return
 
-        ops = sum(v.ops for v in self.input.values())
+        ops = sum(v.ops for v in self.input.values() if not np.isnan(v.ops))
```

Comment:
@FabioLuporini it's unrelated to Cast but it's just a one line change so put it there. `v.ops` can be NaN in some cases with compression and it breaks the tuning and performance output.

Reply:
sure

Reply:
Can this be related to traffic? On top of my head, maybe we can get rid to any check in lines 462+..?